Makefile

```diff
@@ -74,8 +73,7 @@ ifneq ($(BUILDKIT_CONTAINER_RUNNING),$(BUILDKIT_CONTAINER_NAME))
 		--privileged \
 		-p 1234:1234 \
 		$(BUILDKIT_IMAGE) \
-		--addr $(BUILDKIT_HOST) \
-		--allow-insecure-entitlement security.insecure
+		--addr $(BUILDKIT_HOST)
```

Comment:
We don't need `--addr`, buildkit will respect `BUILDKIT_HOST` env var, but we can clean this up later.